vitrina/datasets/models.py

```diff
@@ -226,6 +226,13 @@ def tag_name_array(self):
     def category_title(self):
         return self.category.title if self.category else ""
 
+    def management_area(self):
+        if self.organization:
+            if not self.organization.is_root():
+                if self.organization.get_root().get_children_count() > 1:
+                    return self.organization.get_root().pk
```

Comment:
I'm not sue if `get_root` is an expensive call with a query to database, but it would be safer if only one call would be made and result would be assigned to a variable.